kolibri/plugins/facility/assets/src/views/ManageClassPage/CopyClassSidePanel.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+
+  <div>
+    <SidePanelModal
+      alignment="right"
+      sidePanelWidth="700px"
+      closeButtonIconType="close"
+      @closePanel="closeSidePanelModal"
+    >
+      <template #header>
+        <h1 class="side-panel-title">{{ copyClasslabel$() }}</h1>
+      </template>
+      <template #default>
+        <div>
+          <KTextbox
+            v-model="copiedClassName"
+            type="text"
+            :label="classTitleLabel$()"
+            :autofocus="true"
+            :maxlength="120"
```

Comment:
This should be 100 to match the API (I tried making a max-length class name and got an API error)